c/call-bind/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
same comment as above - the LICENSE can just be copied over from the base folder